### DISCUSSION TITLE
Visualizzare i file superiori ad una certa dimensione

### DISCUSSION THREAD

**Discussion by paolomorabito** [indomus/forum#1657] [HUB: Home Assistant]
Ho HASSIO installato su una VM. Man mano che il tempo passa lo spazio occupato aumenta e mi ritrovo sempre ad aumentare la dimensione della VM. Volevo sapere se c'è un modo per analizzare tutti i file di HASSIO (anche quelli di sistema) e vedere quali sono quelli che superano una certa dimensione, in modo da capire il motivo di questo aumento sconsiderato. 

Tengo presente che il db principale di Hass è di circa 5 GB e influxdb di 90 MB (stando a quanto scritto [qui](https://github.com/home-assistant/core/issues/21715). Ovviamente dubito che il file di influx sia tale, quindi vorrei indagare meglio e capire se, oltre a questo, potrebbero esserci dei file (di log etc) che occupano spazio inutile.

**Comment by 50494554524F** [comment]
i file sono tutti nelle directory accessibili con l'add-on samba... se invece intendi il file della vm probabilmente hai impostato un hdd a dimensione dinamica ed è normale, non serve aumentare la dimensione dello hdd

**Reply by paolomorabito** [reply]
Ho samba, ma i file non sono tutti lì. Per esempio, i file db di influx non ci sono. Se faccio la somma di tutti i file accessibili da samba arrivo a circa 8 Giga (che comprendono il db di HA, i file di configurazione, log, i file di backup etc..), mentre attualmente HA mi segna che su 48 GB di file system ne sta occupando 45. E mi dice che sta raggiungendo il limite. Quindi deve esserci altro, da qualche altra parte

**Reply by 50494554524F** [reply]
hai rimosso anche le righe sotto  "veto_files:" nella configurazione di samba?!

**Reply by paolomorabito** [reply]
Sì, e vedo sempre le stesse cartelle/files. Dove sbaglio?

**Reply by paolomorabito** [reply]
Che poi, anche nella stessa descrizione dell'add-on è scritto che permette di vedere solo i file di configurazione di HA. I file di influxdb, per esempio, non sono dentro questa cartella/sottocartella
